src/DependencyInjection/Configuration.php

```diff
@@ -74,4 +66,14 @@ private function addPlugins(ArrayNodeDefinition $rootNode): void
         ->end()
         ;
     }
+
+    private function getRootNode(TreeBuilder $treeBuilder): ArrayNodeDefinition
+    {
+        if (method_exists($treeBuilder, 'getRootNode')) {
+            /** @phpstan-ignore-next-line */
+            return $treeBuilder->getRootNode();
+        }
```

Comment:
Becomes useless since Sylius 1.8 required Symfony ^4.4.
Can be in another PR if you prefer.